test/unit/node/util.test.ts

```diff
@@ -447,6 +447,33 @@ describe("onLine", () => {
 
     expect(await received).toEqual(expected)
   })
+
+  describe("used with a process missing stdout ", () => {
+    it("should throw an error", async () => {
+      // Initialize a process that does not have stdout.
+      // "If the child was spawned with stdio set to anything
+      // other than 'pipe', then subprocess.stdout will be null."
+      // Source: https://stackoverflow.com/a/46024006/3015595
+      // Other source: https://nodejs.org/api/child_process.html#child_process_subprocess_stdout
+      // NOTE@jsjoeio - I'm not sure if this actually happens though
+      // which is why I have to set proc.stdout = null
+      // a couple lines below.
+      const proc = cp.spawn("node", [], {
+        stdio: [null],
```

Comment:
Does

```
stdio: [0, "empty"]
```
work?

Reply:
![image](https://user-images.githubusercontent.com/3806031/142923303-2fd218f7-b7ee-4b39-996e-051a9717f0bd.png)

Looks like TS doesn't like this but `stdio: 'ignore'` worked!

Reply:
Nice!